settings.gradle

```diff
@@ -13,4 +13,5 @@ findProject(':adapter:validator')?.name = 'validator'
 include 'application'
 include 'application:cli-app'
 findProject(':application:cli-app')?.name = 'cli-app'
-
+include 'adapter:exporter'
```

Comment:
To investigate

Reply:
It seems like this comes from a bad merge. Has to be deleted.

But when merging the master with changes from the last release this comes back in the code.